src/components/Settings.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from "react";
+import { createContext } from "react";
+import { useState } from "react";
+import ReactSwitch from "react-switch";
+
+
+const Settings = (props) => {
+
+     const [theme, setTheme] = useState(props.text);
+    const [colorTheme, setColorTheme] = useState(props.text);
+    const toggleTheme = () => {
+        setTheme((curr) => (curr === "aqua" ? "darkblue" : "aqua"));
+    }
+     useEffect(()=>{
+       document.body.style.backgroundColor = theme;
+    },[theme]);
+
```

Comment:
Currently, when the Settings component is rendered for the first time, the `Change Background Color` button needs to be clicked twice to change the background color. This issue occurs because `props.text` is initially an empty string. I suggest modifying the logic so that the `theme` state has a default value (e.g., `const [theme, setTheme] = useState("aqua");`). Then you could also remove the text prop because you are handling the theme only in the Settings component.